### PR TITLE
Incron setup

### PR DESCRIPTION
**Pull request by jriedl**

### CODE REVIEW

File: cpp/data/pipeline_script_triggers [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/var/tmp/ IN_CREATE,IN_DELETE /usr/local/bin/trigger_pipeline_script.sh $@$# $%
```
Comment (ruschein) [inline cpp/data/pipeline_script_triggers:1]: Any reason why this is indented?

File: cpp/elasticsearch/extract_existing_fulltext_ids.sh [hunk 2]
```diff
@@ -31,6 +31,18 @@ function GetDownloadStats() {
     echo -n $(echo "$response" | wc -c) >&2
     echo " bytes" >&2
     echo "------------------------------------" >&2
+    error=$(CheckError $response)
+    if [[ $(echo -n "$error" | wc --chars) -ne 0 ]]; then
```
Comment (ruschein) [inline cpp/elasticsearch/extract_existing_fulltext_ids.sh:35]: Should we also send an error email?

File: cpp/elasticsearch/extract_existing_fulltext_ids.sh [hunk 3]
```diff
@@ -43,7 +55,7 @@ fi
 id_output_file=$1
 
 echo "Checking write access to output file..."
-> $id_output_file
+[ -w "$id_output_file" ] || (echo "Cannot write to to $id_output_file" && exit 1)
```
Comment (ruschein) [inline cpp/elasticsearch/extract_existing_fulltext_ids.sh:58]: Any reason you perform the echo and exit in a subshell?  The main shell will continue...

File: cpp/pipelines/enable_pipeline_triggers.sh [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+if [[ -f /etc/redhat-release ]]; then
+    systemctl start incrond
+elif [[ -f /etc/debian_version ]]; then
+    systemctl start incron
+else
+    echo "Could not detect Linux distribution"
+    exit 1
+fi
+
```
Comment (ruschein) [inline cpp/pipelines/enable_pipeline_triggers.sh:22]: Why the blank lines at the end?

File: cpp/data/pipeline_script_triggers (hunk 1 above)
Comment (jriedl) [reply cpp/data/pipeline_script_triggers:1]: This seems to be a display problem. There is no indentation in the file.

File: cpp/elasticsearch/extract_existing_fulltext_ids.sh (hunk 2 above)
Comment (jriedl) [reply cpp/elasticsearch/extract_existing_fulltext_ids.sh:35]: The script is normally called from within a cronjob that should fail if a subscript fails.

File: cpp/elasticsearch/extract_existing_fulltext_ids.sh (hunk 3 above)
Comment (jriedl) [reply cpp/elasticsearch/extract_existing_fulltext_ids.sh:58]: Like this it is a oneliner. The main shell will also exit due to the -o errexit flag

Comment (ruschein) [reply cpp/elasticsearch/extract_existing_fulltext_ids.sh]: It's also an ugly hack and relies on the reader thinking about the errexit flag.  Generally we want code to be "obvious" and locally comprehensible, if possible.

**Pull request merged**